app/views/schedules/_event.html.haml

```diff
@@ -4,11 +4,15 @@
       = image_tag speaker.gravatar_url, :class => "img-circle pull-right all-speaker-pic", |
                                         :alt => speaker.name, |
                                         :title => speaker.name |
-
     %p
       = canceled_replacement_event_label(event, event_schedule)
       = replacement_event_notice(event_schedule)
-
+    - if current_user
+      = link_to('#', onClick: 'starClicked();') do
```

Comment:
Why not using a jQuery clickhandler and binding it to an id instead of inline js?